### ISSUE TITLE
AO 2019-01 missing citations

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/openFEC#3669]
https://www.fec.gov/data/legal/advisory-opinions/2019-01/

The “cited in this AO” should appear immediately, and the “this AO is cited by” should appear after the weekly reload (should have appeared for this one last weekend)

**Comment by lbeaufort** [comment]
Looks like AO 2019-01 is missing OCR text, along with others. I emailed Salient to see if they can get the OCR text to appear. 

Also, it looks like other  AO's are missing OCR text for their final opinions:

| ao_no   | ocrtext   |
|---------|-----------|
| 2018-06 | NULL    |
| 2018-15 | NULL    |
| 2019-01 | NULL    |
| 2019-02 |NULL    |
| 2019-03 |NULL    |

```
SELECT ao_no, ocrtext FROM aouser.document
INNER JOIN aouser.ao USING (ao_id)
WHERE category = 'Final Opinion'
AND ocrtext IS NULL order by ao_no
```

**Comment by pkfec** [comment]
Waiting to hear from salient to get back to us and tell y the OCR texts are missing from their system.

**Comment by lbeaufort** [comment]
I identified some others that were missing OCR text, but weren't `NULL` and Salient fixed those as well. These were harder to find- they weren't `' '` so I'm not sure how they were saved. I queried the length of the OCR text to find them: 
```
SELECT ao_no, max(char_length(ocrtext))
FROM aouser.document
INNER JOIN aouser.ao USING (ao_id)
WHERE category = 'Final Opinion'
group by ao_no
having max(char_length(ocrtext)) < 100
order by ao_no desc 
```

Now that all AO's have OCR text, I did a global reload so the "cited by" citations would be populated as well. 

`cf run-task api "python manage.py refresh_current_legal_docs_zero_downtime" -m 4G --name refresh-legal-20190403`
